webpack/assets/javascripts/react_app/components/FactCharts/FactChartActions.js

```diff
@@ -1,23 +1,15 @@
 import {
-  FACT_CHART_REQUEST,
-  FACT_CHART_SUCCESS,
-  FACT_CHART_FAILURE,
+  FACT_CHART,
   FACT_CHART_MODAL_OPEN,
   FACT_CHART_MODAL_CLOSE,
 } from './FactChartConstants';
 
-import { ajaxRequestAction } from '../../redux/actions/common/index';
-
-export const getChartData = (url, id) => dispatch =>
-  ajaxRequestAction({
-    dispatch,
-    requestAction: FACT_CHART_REQUEST,
-    successAction: FACT_CHART_SUCCESS,
-    failedAction: FACT_CHART_FAILURE,
-    url,
-    item: { id },
-  });
-
+export const getChartData = (url, id) => ({
+  type: 'API_GET',
+  subtype: FACT_CHART,
```

Comment:
maybe it would be better to call it just `type` ?

Reply:
"Actions must have a type property that indicates the type of action being performed" -
the action that is preformed is API_GET, not FACT_CHART
So the `type: 'API_GET',` should stay but maybe can you suggest a different name for `subtype`?

Reply:
What about renaming `subtype` to `dispatchType` or `outputType`?